lib/manageiq/environment.rb

```diff
@@ -0,0 +1,108 @@
+require 'fileutils'
+require 'pathname'
+
+module ManageIQ
+  module Environment
+    APP_ROOT = Pathname.new(File.expand_path('../../..', __FILE__))
```

Comment:
I never liked this style of using File methods before the Pathname...Pathname has those methods arleady.   Can we do this instead, as I think it's more readable?  (I do realize this was there beforehand)

```ruby
APP_ROOT = Pathname.new(__dir__).join("../..")
```